plugins/scaffolder-backend/src/scaffolder/actions/builtin/debug/log.ts

```diff
@@ -45,9 +45,6 @@ export function createDebugLogAction() {
             title: 'List all files in the workspace, if true.',
             type: 'boolean',
           },
-          extra: {
-            title: 'Extra info',
```

Comment:
extra is actually part of the stringified output in the first line of handler.
It's meant as an open field to add additional debug information.

Reply:
Which also has the benefit of just being untyped too, so you can log anything from any other action 🙏